pandas/io/formats/printing.py

```diff
@@ -280,8 +281,10 @@ def format_object_summary(obj, formatter, is_justify=True, name=None):
         string formatter for an element
     is_justify : boolean
         should justify the display
-    name : name, optiona
+    name : name, optional
         defaults to the class name of the obj
+    is_multi : bool, default False
```

Comment:
this is not going to be acceptable
this cannot know anything about a MultiIndez
you can override the formatters in multi if you really really need

Reply:
Can it be called ``line_break_on_values``?

Reply:
yes ``line_break_one_values`` is fine.  This function just cannot reference any pandas internal things.